examples/mcp/async_loop.py

```diff
@@ -0,0 +1,52 @@
+import asyncio
+import logging
+import os
+import time
+
+import aiohttp
+import hydra
+from omegaconf import DictConfig
+
+from examples.gaia_agent.eval import load_dataset, task_to_observations
+from examples.gaia_agent.steps import GaiaTape
+from tapeagents.io import save_json_tape
+from tapeagents.orchestrator import EnvironmentManager, execute_with_env
+
+logger = logging.getLogger(__name__)
+
+
+@hydra.main(
+    version_base=None,
+    config_path="../../conf",
+    config_name="async_mcp",
+)
+def main(cfg: DictConfig) -> None:
+    asyncio.run(amain(cfg))
+
+
+async def amain(cfg: DictConfig) -> None:
+    cfg.llm.base_url = os.environ["BASE_URL"]
```

Comment:
One normally does this kind of configuration with Hydra, but here you are overriding this path. At least we should make using the env variable optional.

Reply:
It is done on purpose to not to include internal URLs in the config